### ISSUE TITLE
Unit tests ignore `xunit.runner.json` - Explains why the new `stopOnFail` setting wasn't working

### ISSUE THREAD

**Issue by tig** [gui-cs/Terminal.Gui#2769]
See https://github.com/xunit/visualstudio.xunit/issues/189

Fix is to set `xunit.runner.json` to Copy On Newer:

![image](https://github.com/gui-cs/Terminal.Gui/assets/585482/b7a81d79-a4fc-4b8d-b104-324946209bbc)

(Adding this to `UnitTests.csproj`:

```xml
  <ItemGroup>
    <Content Include="xunit.runner.json" CopyToOutputDirectory="PreserveNewest" />
  </ItemGroup>
```

We can remove this line from `./UnitTests/AssemblyInfo.cs` (it WAS what was preventing tests from running in parallel, but is no longer needed if `xunit.runner.json` is working):

```cs
// Since Application is a singleton we can't run tests in parallel
[assembly: CollectionBehavior (DisableTestParallelization = true)]
```

Once this is done, stopOnFail can be enabled by adding `  "stopOnFail": true` to `xunit.runner.json`.

I'd like to know if anyone has an objection to setting `stopOnFail` to true. I'd DEF want this for CI/CD, but am not sure if I like it for dev work.

**Comment by BDisp** [comment]
There is situations which is preferable the unit tests stooped instead of some never end.

**Comment by tig** [comment]
> There is situations which is preferable the unit tests stooped instead of some never end.

Sorry @BDisp but I can't parse what you're saying? Can you try again?

**Comment by BDisp** [comment]
> Sorry @BDisp but I can't parse what you're saying? Can you try again?

I was referring that some unit tests fail silent and only opening the Tests on the drop down of the Output window we can verify there is an exception. But never mind.

**Comment by BDisp** [comment]
Well I also agree with @tznind. If the intention is stop the tests when one of them fails, then we stop al  tests. I know running the tests in a command line the failed tests is only show when they finished, but we can do Ctrl+C to stop all. For CI/CD the @tznind suggestion is a good choice.

**Comment by tig** [comment]
Cool. That's how we'll address this then.

**Comment by tig** [comment]
I've also fixed this in #2612 so it will be picked up for v2 when I merge that.
